tools/cli/workflow.go

```diff
@@ -453,60 +469,68 @@ func newBatchCommands() []cli.Command {
 			Name:  "start",
 			Usage: "Start a batch operation job",
 			Flags: []cli.Flag{
-				cli.StringFlag{
-					Name:  FlagListQueryWithAlias,
-					Usage: "Query to get workflows for being executed this batch operation",
+				&cli.StringFlag{
+					Name:    FlagListQuery,
+					Aliases: []string{"q"},
+					Usage:   "Query to get workflows for being executed this batch operation",
 				},
-				cli.StringFlag{
-					Name:  FlagReasonWithAlias,
-					Usage: "Reason to run this batch job",
+				&cli.StringFlag{
+					Name:    FlagReason,
+					Aliases: []string{"re"},
+					Usage:   "Reason to run this batch job",
 				},
-				cli.StringFlag{
-					Name:  FlagBatchTypeWithAlias,
-					Usage: "Types supported: " + strings.Join(batcher.AllBatchTypes, ","),
+				&cli.StringFlag{
+					Name:    FlagBatchType,
+					Aliases: []string{"bt"},
+					Usage:   "Types supported: " + strings.Join(batcher.AllBatchTypes, ","),
 				},
 				// below are optional
-				cli.StringFlag{
-					Name:  FlagSignalNameWithAlias,
-					Usage: "Required for batch signal",
-				},
-				cli.StringFlag{
-					Name:  FlagInputWithAlias,
-					Usage: "Optional input of signal",
-				},
-				cli.StringFlag{
-					Name:  FlagSourceClusterWithAlias,
-					Usage: "Required for batch replicate",
-				},
-				cli.StringFlag{
-					Name:  FlagTargetClusterWithAlias,
-					Usage: "Required for batch replicate",
-				},
-				cli.IntFlag{
+				&cli.StringFlag{
+					Name:    FlagSignalName,
+					Aliases: []string{"sn"},
+					Usage:   "Required for batch signal",
+				},
+				&cli.StringFlag{
+					Name:    FlagInput,
+					Aliases: []string{"in"},
+					Usage:   "Optional input of signal",
+				},
+				&cli.StringFlag{
+					Name:    FlagSourceCluster,
+					Aliases: []string{"sc"},
+					Usage:   "Required for batch replicate",
+				},
+				&cli.StringFlag{
+					Name:    FlagTargetCluster,
+					Aliases: []string{"tc"},
+					Usage:   "Required for batch replicate",
+				},
+				&cli.IntFlag{
 					Name:  FlagRPS,
 					Value: batcher.DefaultRPS,
 					Usage: "RPS of processing",
 				},
-				cli.BoolFlag{
+				&cli.BoolFlag{
 					Name:  FlagYes,
 					Usage: "Optional flag to disable confirmation prompt",
 				},
-				cli.IntFlag{
+				&cli.IntFlag{
 					Name:  FlagPageSize,
 					Value: batcher.DefaultPageSize,
 					Usage: "PageSize of processiing",
 				},
-				cli.IntFlag{
+				&cli.IntFlag{
 					Name:  FlagRetryAttempts,
 					Value: batcher.DefaultAttemptsOnRetryableError,
 					Usage: "Retry attempts for retriable errors",
 				},
-				cli.IntFlag{
-					Name:  FlagActivityHeartBeatTimeoutWithAlias,
-					Value: int(batcher.DefaultActivityHeartBeatTimeout / time.Second),
-					Usage: "Heartbeat timeout for batcher activity in seconds",
+				&cli.IntFlag{
+					Name:    FlagActivityHeartBeatTimeout,
+					Aliases: []string{"hbt"},
+					Value:   int(batcher.DefaultActivityHeartBeatTimeout / time.Second),
+					Usage:   "Heartbeat timeout for batcher activity in seconds",
```

Comment:
we really need a "DurationFlag" 😢 
though we'd probably have to handle `123` == `123s` upgrading.